02-overview-surveys.Rmd

```diff
@@ -1,11 +1,12 @@
 # Overview of Surveys {#c02-overview-surveys}
```

Comment:
Another thought - since this chapter focuses on the Survey Life Cycle, how about a title that's more explicit like "Overview of the Complex Survey Lifecycle"?

Reply:
Also, have to pick one or two words for life cycle 😂

Reply:
Did some digging - should be two words 👍